pcf/test/particle/aws/test_particles.py

```diff
@@ -24,6 +24,7 @@
 from pcf.particle.aws.kms.kms_key import KMSKey
 from pcf.particle.aws.s3.s3_bucket import S3Bucket
 from pcf.particle.aws.vpc.vpc_instance import VPCInstance
+from pcf.particle.aws.ecr.ecr_repository import ECRRepository
```

Comment:
pretty sure you dont need to add particles since they are imported dynamically.

Reply:
Removed all the unused imports